server/container/guice/guice-common/src/main/java/org/apache/james/modules/CommonServicesModule.java

```diff
@@ -57,11 +55,11 @@ protected void configure() {
         install(new DNSServiceModule());
         install(new AsyncTasksExecutorModule());
         install(new DropWizardMetricsModule());
+        install(new PojoProbesModule());
```

Comment:
looks like a bad idea : you are requiring every servers that include CommonServices to Sieve, Mailbox, etc.
It's exactly what I tried to avoid